tests/test_inference.py

```diff
@@ -287,10 +287,10 @@ def test_bestfit_tf(xes):
     guess['er_rate_multiplier'] = xs[np.argmin(ys)]
     assert len(guess) == 2
 
```

Comment:
`import tensorflow_probability as tfp` and`from iminuit import Minuit` are not used anymore in the tests. We can delete them.